volume/service/store.go

```diff
@@ -195,11 +195,11 @@ func (s *VolumeStore) purge(ctx context.Context, name string) error {
 	if exists {
 		driverName := v.DriverName()
 		if _, err := s.drivers.ReleaseDriver(driverName); err != nil {
-			log.G(ctx).WithError(err).WithField("driver", driverName).Error("Error releasing reference to volume driver")
+			log.G(ctx).WithFields(log.Fields{"error": err, "driver": driverName, "volume-name": name}).Error("Error releasing reference to volume driver")
```

Comment:
What was the problem with `WithError`?

Reply:
Mostly because `WithError`, `WithField`, `WithFields` do a lot under the hood; they have to merge new values with old values, and performing reflection; doing a single `WithFields` can avoid doing that multiple times; https://github.com/moby/moby/blob/921ac59a739777d51d7af62484b39b1499bc396f/vendor/github.com/sirupsen/logrus/entry.go#L105-L151

Reply:
@AkihiroSuda ☝️ does that answer your question? Do the changes LGTY?